iris_ugrid/tests/integration/test_synthetic_data.py

```diff
@@ -0,0 +1,87 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Integration tests for the
```

Comment:
I think this description is a copy-paste from the test_ugrid_load.py file and needs updating

Reply:
Oops.
Good spot !